### PR TITLE
Review project

### PR DESCRIPTION
**Pull request by engrabdullahriaz**
added code by grace to review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <h1 id="title">OUR SURVEY FORM</h1>
```
Comment (engrabdullahriaz) [inline index.html:12]: Please use the header element in h1 for the best practice of semantic tags of HTML.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
```
Comment (engrabdullahriaz) [inline index.html:11]: Please avoid unnecessary blank space for practicing best HTML practice.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      <option value="country"></option>
+
+      <option value="time zone"></option>
+    </select>
+    <br>
+    <input type="radio" name="gender" value="1"> <label>Male</label>
+    <input type="radio" name="gender" value="2"> <label>FEMALE</label>
+    <input type="radio" name="gender" value="3"> <label>TRANSGENDER</label>
+    <input type="radio" name="gender" value="4"> <label>BIGENDER</label>
```
Comment (engrabdullahriaz) [inline index.html:36]: Please avoid using uppercase for naming instead use lowercase or capitalize it

File: index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    <br>
+    <input type="submit" id="submit">
+  </form>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
+  <script src="js/index.js"></script>
+</body>
+
+</html>
```
Comment (engrabdullahriaz) [inline index.html:53]: I kindly suggest that you add linters to your file for easy catching syntax errors and produce more efficient code

File: style.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  
+  #title {
```
Comment (engrabdullahriaz) [inline style.css:7]: Please avoid using id selector instead use class selectors for CSS best practices.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    border-radius: 7px;
+  }
+  
+  #submit {
+    background-color: aqua;
+    color: white;
+    padding: 0.4rem;
+    cursor: pointer;
+    font-size: 1.15rem;
+  }
```
Comment (engrabdullahriaz) [inline style.css:33]: Please include linters for easy debugging.

File: index.html (hunk 3 above)
Comment (omar-muhamad) [reply index.html:36]: - [x] kindly ask for the changes you want directly in a polite way like you did for the rest of your review comments and kindly avoid using the word `I think`. So please edit this comment.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @engrabdullahriaz,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You used inline comments.
- You mentioned the reason why the reviewee had to do the required changes.
- Mostly you asked for changes in a polite way.

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the replies to your inline comments.

## [Optional] suggestions:

- [x] Next time please make your inline comments inside one review instead of making it in multiple reviews.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @engrabdullahriaz,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**